styleguide/derek/incubation/homepage.ejs

```diff
@@ -52,20 +42,21 @@
 <%- partial("../pattern-components/tabs/tabs-horizontal.ejs") %>
 
 
-  <!-- Quadrant -->
-  <div class="container-fluid quadrant" id="quadrant">
-    <div class="container">
-      <div class="quadrant-row">
-        <div class="quadrant-textContainer">
-          <h2 class="quadrant-title">Fanatical Support wherever you are, day or night, we have you covered 24x7x365</h2>
-          <p class="quadrant-text">We're more than just a hosting provider - we're your trusted service partner, and we're ready to help solve the toughest problems. Gartner has recognized us as a leader in their <a href="[rsweb:about:cloud-enabled]">Magic Quadrant for Cloud Enabled Managed Hosting.</a></p>
-            <div class="quadrant-buttonContainer">
-              <a class="button learning" href="[rsweb:why-rackspace]">Learn More</a>
-            </div>
+<!-- Case Study -->
+
+<div class="caseStudy-wrapper">
+  <div class="caseStudy-container">
+    <div class="caseStudy-row">
+      <div class="caseStudy-content">
+        <img class="caseStudy-image" src="/imgs/derek/logos/mckesson_logo.png"/>
```

Comment:
Is it just me or is the McKesson logo blurry?

![image](https://user-images.githubusercontent.com/5263371/44418209-5dcb6900-a53d-11e8-9469-eee8e9673a2a.png)

Reply:
Yeah a little bit. I don't have a higher quality logo. Thats the one I got in the task, it's more of a question for Mike like hey do we want a smaller one or do we want to source a better logo. It would be switched out in the CMS anyways so it's not the hard and fast logo.